ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -143,6 +143,31 @@ def test_run_ipahealthcheck_list_source(self):
         for source in sources:
             assert source in result.stdout_text
 
+    def test_human_output(self):
+        """
+        Test that in human output the severity value is correct
+
+        Only the SUCCESS (0) value was being translated, otherwise
+        the numeric value was being shown (BZ 1752849)
+        """
+        self.master.run_command(["systemctl", "stop", "sssd"])
+        result = self.master.run_command(
+            [
+                "ipa-healthcheck",
+                "--source",
+                "ipahealthcheck.meta.services",
+                "--check",
+                "sssd",
+                "--output-type",
+                "human",
+            ],
+            raiseonerr=False,
+        )
+        assert result.returncode == 1
+        assert result.stdout_text.strip() == \
+            "ERROR: ipahealthcheck.meta.services.sssd: sssd: not running"
```

Comment:
IMO, this assertion is very brittle, because the error message may be changed(?) in `ipa-healthcheck` unless it's guaranteed to be so. May be regular exprs?

Reply:
It's a sister project so hopefully we'll communicate. It isn't impossible for the string to change but is not very likely.